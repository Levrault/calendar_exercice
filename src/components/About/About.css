@keyframes FadeIn {
  from {
    opacity: 0;
  }
  
  to {
    opacity: 1;
  }
}

.container {
  animation-duration: .5s;
  animation-name: FadeIn;
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.maintitle {
  color: $palette-title;
  font-weight: 300;
  margin: $space-l auto;
}

.title {
  color: $palette-title;
  font-weight: 100;
}
