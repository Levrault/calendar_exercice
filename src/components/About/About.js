import React from 'react';
import styles from './About.css';

/**
 * About page
 */
const About = () => (
  <div className={styles.container}>
    <h1 className={styles.maintitle}>About</h1>
    <p>This is small calendar app created has an exercice</p>

    <h2 className={styles.title}>Stack</h2>
    <p>It uses the following technologies</p>
    <ul>
      <li>React</li>
      <li>React-Redux</li>
      <li>Redux-thunk</li>
      <li>Reselect</li>
      <li>Jest with enzyme</li>
      <li>Deployd</li>
      <li>Storybook</li>
    </ul>

    <h2 className={styles.title}>How to use</h2>
    <p>Yarn start compile project and start deployd</p>
    <p>Yarn jest will execute all the text</p>
    <p>Yarn storybook will compile stories</p>

    <h2 className={styles.title}>Icon pack</h2>
    <a href="https://material.io/tools/icons/?style=baseline" target="_blank">
      Material icon pack
    </a>
  </div>
);

export default About;
