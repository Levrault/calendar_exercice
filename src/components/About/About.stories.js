import React from 'react';
import { storiesOf } from '@storybook/react';
import ContentWrapper from '../../tools/storybook/ContentWrapper';
import About from './About';

storiesOf('About', module)
  .addDecorator(ContentWrapper)
  .add('default', () => (
    <About />
  ));
