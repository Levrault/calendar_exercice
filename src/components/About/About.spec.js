import React from 'react';
import renderer from 'react-test-renderer';
import About from './About';

describe('About component', () => {
  it('should render without problem', () => {
    const component = renderer.create(
      <About />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
