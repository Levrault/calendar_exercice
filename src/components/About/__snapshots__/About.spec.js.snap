// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render without problem 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    This is small calendar app created has an exercice
  </p>
  <h2>
    Stack
  </h2>
  <p>
    It uses the following technologies
  </p>
  <ul>
    <li>
      React
    </li>
    <li>
      React-Redux
    </li>
    <li>
      Redux-thunk
    </li>
    <li>
      Reselect
    </li>
    <li>
      Jest with enzyme
    </li>
    <li>
      Deployd
    </li>
    <li>
      Storybook
    </li>
  </ul>
  <h2>
    How to use
  </h2>
  <p>
    Yarn start compile project and start deployd
  </p>
  <p>
    Yarn jest will execute all the text
  </p>
  <p>
    Yarn storybook will compile stories
  </p>
  <h2>
    Icon pack
  </h2>
  <a
    href="https://material.io/tools/icons/?style=baseline"
    target="_blank"
  >
    Material icon pack
  </a>
</div>
`;
